vhdbuilder/scripts/automate_version_bump.sh

```diff
@@ -17,24 +17,18 @@ build_ids=$3
 branch_name=imageBump/$new_image_version
 pr_title="VersionBump"
 
-# This function finds the current SIG Image version from pkg/agent/datamodelosimageconfig.go
+# This function finds the current SIG Image version from the input JSON file
 find_current_image_version() {
     filepath=$1
-    while read -r p; do
-        if [[ $p == *"LinuxSIGImageVersion"* ]]; then
-            current_image_version=$(echo $p | awk -F'\"' '{print $2}')
-            echo "Image version is $current_image_version, cut from line $p"
-            break
-        fi
-    done < $filepath
+    current_image_version=$(jq .version $filepath)
```

Comment:
not sure if this is used elsewhere, but might want to tack `-r` on here so we get rid of the surrounding quotes

Reply:
Thanks, updated.